swift/aperture/util.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+private final class StandardErrorOutputStream: TextOutputStream {
+  func write(_ string: String) {
+    FileHandle.standardError.write(string.data(using: .utf8)!)
```

Comment:
Well probably not in the context of this app, but it is possible to create strings that can not be UTF8 encoded.
IMO its can be okay to force unwrap in situations like this, but many people prefer never to force unwrap optionals.

In general you should think "What do I want to happen if the string can't be converted to utf8 encoded data?" 🤔

**Answer:** "I want to print out an alternative error message"
```swift
FileHandle.standardError.write(string.data(using: .utf8) ?? "Unknown error")
```

**Answer:** "I don't want to print anything"
```swift
if let data = string.data(using: .utf8) {
    FileHandle.standardError.write(data)
}
```

**Answer:** "Should never happen.. I want the app to crash.."
```swift
guard let data = string.data(using: .utf8) else {
    fatalError("Cant create utf8 encoded data from string: \(string)")
}
FileHandle.standardError.write(data)
```
This last on is a lot more verbose than just force unwrapping but gives better error messages.